apis/apiextensions/v1beta1/zz_generated.deepcopy.go

```diff
@@ -284,16 +284,16 @@ func (in *ContainerFunction) DeepCopyInto(out *ContainerFunction) {
 		*out = new(v1.PullPolicy)
 		**out = **in
 	}
-	if in.Timeout != nil {
-		in, out := &in.Timeout, &out.Timeout
-		*out = new(metav1.Duration)
-		**out = **in
-	}
 	if in.ImagePullSecrets != nil {
 		in, out := &in.ImagePullSecrets, &out.ImagePullSecrets
 		*out = make([]v1.LocalObjectReference, len(*in))
 		copy(*out, *in)
 	}
+	if in.Timeout != nil {
```

Comment:
this change looks like no-op, the if block was removed from line 287 and added back to line 292. Can we revert it?

Reply:
This is a generated file. No idea why the deepcopy code generator decided to make this re-ordering but I think we should let it. If we don't it's just going to try make this change again next time someone runs `go generate`.